tests/compilation/test_compile_empty.py

```diff
@@ -0,0 +1,17 @@
+import tempfile
+
+import pytest
+
+from solc import (
+    compile_files,
+)
+
+from solc.exceptions import ContractsNotFound
+
+
+def test_compile_empty_folder():
+    """Execute compile on a folder without contracts."""
+
+    with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
I think this isn't python2 compatable.

You can do this with the `tmpdir` fixture provided by pytest.

```python
def test_things(tmpdir):
    dir_that_will_be_auto_cleaned_up = str(tmpdir.mkdir('some-dir'))
    ...  # do things.
```